lib/client-assets.php

```diff
@@ -791,9 +791,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'colors' => $color_palette,
 	);
 
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
-		. 'wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
-		. '} );'
+	wp_add_inline_script(
+		'wp-editor',
+		(
+			'jQuery.when( wp.api.init(), wp.api.init( { versionString: \'gutenberg/v1/\' } ) ).done( function() {'
```

Comment:
Do we need to initialize twice like this?

`jQuery` is not an explicit dependency of `wp-editor` and should be added as one if we're going to call on it. Currently it works likely because it's incidentally a dependency of another script on the page, but we can't rely on it to exist this way.

Reply:
`wp_add_inline_script` doesn't seem to support specifying dependencies and so I've added a manual check for jQuery in 3d1b66c1.

> Do we need to initialize twice like this?

I'm... not sure.

I couldn't find any documentation about this stuff. [The source](https://github.com/WP-API/client-js/blob/master/js/load.js#L367) seems to imply that `init()` is to be called once for every API root and version string. In our case, we have two version strings: `/wp/v2` and `/gutenberg/v1`. I'd love for someone who knows more about the WP API client to chime in 😅